client/ayon_core/tools/loader/ui/product_group_dialog.py

```diff
@@ -15,13 +16,22 @@ def __init__(self, controller, parent):
         group_name_input = PlaceholderLineEdit(self)
         group_name_input.setPlaceholderText("Remain blank to ungroup..")
 
+        group_picker_btn = QtWidgets.QPushButton()
+        group_picker_btn.setFixedWidth(18)
```

Comment:
Why is this needed?

Reply:
If it is to make it smaller then use rather this:
```
        size_policy = group_picker_btn.sizePolicy()
        size_policy.setVerticalPolicy(
            QtWidgets.QSizePolicy.MinimumExpanding)
        group_picker_btn.setSizePolicy(size_policy)
```

Reply:
It's to avoid this weird size: 
![Uploading image.png…]()


Unfortunately your 'fix' proposed here doesn't solve it.